src/Illuminate/Broadcasting/BroadcastManager.php

```diff
@@ -81,12 +82,20 @@ public function socket($request = null)
             return $request->header('X-Socket-Id');
         }
 
+        if ($request->hasCookie('io')) {
```

Comment:
The original request sent from engine io includes a set cookie header for that. And in rebound I pass the original request headers to the save socket post. This includes the io header which is the socket I'd. 

> On 27 May 2016, at 14:37, Taylor Otwell <notifications@github.com> wrote:
> 
> In src/Illuminate/Broadcasting/BroadcastManager.php:
> 
> > @@ -81,12 +82,20 @@ public function socket($request = null)
> >              return $request->header('X-Socket-Id');
> >          }
> >  
> > +        if ($request->hasCookie('io')) {
> Where is this cookie sent from?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
>